modules/virt-understanding-vm-templates.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-understanding-vm-templates-web_{context}"]
+= Understanding virtual machine templates
+
+Preconfigured Red Hat virtual machine templates are listed in the *Templates* tab within the *Virtualization* page. These  templates are available for different versions of Red Hat Enterprise Linux, Fedora, Microsoft Windows 10, and Microsoft Windows Servers. Each Red Hat virtual machine template is preconfigured with the operating system image, default settings for the operating system, flavor (CPU and memory), and workload type (server).
```

Comment:
Users can select a Red Hat template and save it as a custom template for their use.  i've added this description that I hope will clarify the difference between Red Hat templates and custom templates:

Using a Red Hat template is convenient because the template is already preconfigured. When you select a Red Hat template to create your own custom template, the interactive wizard prompts you to add a boot source if a boot source was not added previously. Then, you can either save your custom template or continue to customize it and save it.

You can also select the interactive wizard directly and create a custom virtual machine template. The wizard prompts you to provide configuration details for the operating system, flavor, workload type, and other settings. You can add a boot source and continue to customize your template and save it.